### PR TITLE
Fix copying test files

### PR DESCRIPTION
**Pull request by simonihmig**
At a previous iteration a regression was introduced, that copied the test files into the test-app, but including their full path:

image

### CODE REVIEW

**Review by NullVoxPopuli — APPROVED** [review]
Since your a maintainer now, can you add a changeset before merge? <3

**Comment by simonihmig** [comment]
> Since your a maintainer now, can you add a changeset before merge?

Sure, done!

**Pull request merged**
